implementation/configuration/src/configuration_impl.cpp

```diff
@@ -1380,6 +1382,64 @@ void configuration_impl::load_trace_filter_match(
     }
 }
 
+int configuration_impl::get_nic(const std::string& _local_address)
+{
+    // creating structures to browse through the available network interfaces
+    struct ifaddrs* ifaddr, *ifa;
+    int family, n;
+    sockaddr_in6* p_addrIP6;
+    std::vector<int> nic(0);    // the network interface card id. Vector in case 
+                    // more than one interface contains the searched IP address
+    struct sockaddr_in6 searched_address;
+    searched_address.sin6_family = AF_INET6;
+    searched_address.sin6_addr = in6addr_any;
+    searched_address.sin6_port = htons( 0xdead );
+    inet_pton( AF_INET6, _local_address.c_str(), &searched_address.sin6_addr ); // actually copying 
+
+    if (getifaddrs(&ifaddr) == -1) // "-1" is an error value
+    {
+        VSOMEIP_ERROR << __func__ <<  
+            strerror(errno) << ::std::endl;
```

Comment:
You need to give a space after the __func__ or you will have the two things together.